frontend/src/global_styles/layout/_main_menu.sass

```diff
@@ -51,6 +51,7 @@ $arrow-left-width: 40px
     +allow-vertical-scrolling
     height: calc(100vh - var(--header-height))
     position: relative
+    border-right: 1px solid var(--main-menu-border-color)
```

Comment:
What is the reason for that change? The commit message says it is for the case that there is no EE token present. However, this is not reflected in the code.

Reply:
It was not correct, the issue was related to the variable 'main-menu-border-width' which was 0px, I set it to 1px for high contrast mode.